homeassistant/components/honeygain/sensor.py

```diff
@@ -0,0 +1,95 @@
+"""Sensors for HoneyGain data."""
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CURRENCY_DOLLAR
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HoneygainData
+from .const import DOMAIN
+
+
+@dataclass
+class SensorValueEntityDescription(SensorEntityDescription):
+    """Class describing Honeygain sensor entities."""
+
+    value: Callable = lambda x: x
+
+
+HONEYGAIN_SENSORS: list[SensorValueEntityDescription] = [
```

Comment:
New integrations/entities must support [translation of names and attributes](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/).

Please adopt your code accordingly.